source/math/parser.py

```diff
@@ -8,10 +8,10 @@
 
 from source.math.math_function import MathFunction
 
-
-# This exception raise when sympy cannot draw a function plot
+STATEMENTS_LIMIT = 15
```

Comment:
No explanation for the meaning of variable.

Reply:
Good point, I'll fix it